demo/index.html

```diff
@@ -47,6 +47,12 @@
     .horizontal-section {
       text-align: center;
     }
+
+    .nounderline {
+      --paper-input-container-underline: {
+        display: none;
```

Comment:
there's a lot of styling options for the internal elements. Added this as it took me a while to find and many people want to shut off the underline on a dropdown.